### PR TITLE
styling with styler

### PR DESCRIPTION
**Pull request by lorenzwalthert**
Testing styler.
Reference: https://github.com/krlmlr/styler/issues/93.

### CODE REVIEW

File: R/compat-lazyeval.R [hunk 1]
```diff
@@ -25,27 +25,27 @@ compat_lazy <- function(lazy, env = caller_env(), warn = TRUE) {
...
+              logical =,
+              integer =,
+              double = {
       if (length(lazy) > 1) {
         warn("Truncating vector to length 1")
         lazy <- lazy[[1]]
       }
       new_quosure(lazy, env)
     },
-    list =
```
Comment (lorenzwalthert) [inline R/compat-lazyeval.R:45]: Not sure about how to continue after `},`. Do you think it's done right now?

**Comment by lorenzwalthert** [comment]
Reference ([#93](https://github.com/krlmlr/styler/issues/93)).

File: tests/testthat/test-unnest.R [hunk 2]
```diff
@@ -82,14 +83,15 @@ test_that(".id creates vector of names for grouped data frame unnest", {
...
-    dplyr::group_by(x)
+  dplyr::group_by(x)
   out <- unnest(df, .id = "name")
 
   expect_equal(out$name, c("a", "b", "b"))
 })
 
 test_that("can use non-syntactic names", {
-  out <- tibble("foo bar" = list(1:2, 3)) %>% unnest()
+  out <- tibble("foo bar" = list(1:2, 3)) %>%
```
Comment (lorenzwalthert) [inline tests/testthat/test-unnest.R:93]: single line pipes not yet supported but issue is open ([#74](https://github.com/krlmlr/styler/issues/74))

File: R/compat-lazyeval.R [hunk 3]
```diff
@@ -25,27 +25,27 @@ compat_lazy <- function(lazy, env = caller_env(), warn = TRUE) {
   }
 
   coerce_type(lazy, "a quosure",
-    formula = as_quosure(lazy, env),
-    symbol = ,
-    language = new_quosure(lazy, env),
-    string = ,
-    character = {
+              formula = as_quosure(lazy, env),
+              symbol =,
```
Comment (lorenzwalthert) [inline R/compat-lazyeval.R:29]: should we add this rlang styling rule that puts a space between `=` and `,`?

**Comment by krlmlr** [comment]
I missed your comments, but we seem to have covered most by now?

File: R/compat-lazyeval.R (hunk 3 above)
Comment (lorenzwalthert) [reply R/compat-lazyeval.R:29]: Done in https://github.com/krlmlr/styler/pull/150.

**Comment by lorenzwalthert** [comment]
This PR was for testing purposes of styler only and I think we can close it.

**Comment by lorenzwalthert** [comment]
Maybe filing a PR to the upstream repo later (with an updated version of styler).

**Pull request closed without merging**